File: Efficiently working with data readers for SQL-Server in VB.NET/[VB]-Efficiently working with data readers for SQL-Server in VB.NET/VB/WindowsApp1/Classes/BaseSqlServerConnections.vb
Public Class BaseSqlServerConnections
    Inherits BaseExceptionsHandler

    ''' <summary>
    ''' This points to your database server
    ''' </summary>
    Protected DatabaseServer As String = "KARENS-PC"
    ''' <summary>
    ''' Name of database containing required tables
    ''' </summary>
    Protected DefaultCatalog As String = ""
    Public ReadOnly Property ConnectionString As String
        Get
            Return $"Data Source={DatabaseServer};" &
                   $"Initial Catalog={DefaultCatalog};Integrated Security=True"
        End Get
    End Property
End Class

File: DataGridView buttons (C#)/[C#]-DataGridView buttons (C#)/C#/TeamBaseLibrary/BaseSqlServerConnections.cs
namespace TeamBaseLibrary
{
    public class BaseSqlServerConnections : BaseExceptionsHandler
    {
        /// <summary>
        /// This points to your database server
        /// </summary>
        protected string DatabaseServer = "KARENS-PC";
        /// <summary>
        /// Name of database containing required tables
        /// </summary>
        protected string DefaultCatalog = "";
        public string ConnectionString
        {
            get
            {
                return $"Data Source={DatabaseServer};Initial Catalog={DefaultCatalog};Integrated Security=True";
            }
        }

    }
}
